src/udev/udev-builtin-path_id.c

```diff
@@ -630,7 +630,12 @@ static int builtin_path_id(sd_device *dev, sd_netlink **rtnl, int argc, char *ar
                         supported_transport = true;
                         supported_parent = true;
                 } else if (streq(subsys, "nvme")) {
-                        const char *nsid;
+                        const char *nsid, *hidden;
+
+                        if (sd_device_get_sysattr_value(dev, "hidden", &hidden) >= 0)
+                                /* NVMe multipath path (GENHD_FL_HIDDEN); ignore */
```

Comment:
maybe add a comment explaining _why_ it is ignored?

Reply:
Force-pushed with an explanation that's a lean version of the commit message. Let me know how that works.